### DISCUSSION TITLE
Pygame_gui is compatible with Linux?

### DISCUSSION THREAD

**Discussion by QuetzalcoutlDev** [MyreMylar/pygame_gui#338] [Q&A] [answered]
I know that in the ISSUES they do not have the option to leave a question, but I would like to know if Pygame_gui is compatible with Linux, because in their documentation it does not tell me if it is, I ask why I know that a Pygame project made in Windows does not go 100% work at Linux or Mac without making some settings

**Comment by lannocc** [comment, marked as answer]
I've been using it in Linux just fine.

**Comment by MyreMylar** [comment]
Hello, there is a discussions area for asking questions.

Yes, the library generally works on desktop flavours of linux. I develop on windows generally but it's all python code and there are only a few little bits of platform specific stuff here and there (including some linux platform specific stuff). The only linux testing I've personally done has been on raspberry Pi computers.

I'll move this over to discussions.
